File: skener.py
R,C,ZR,ZC = map(int,input().split())
article = [input().strip() for _ in range(R)]
matrix = [[0 for i in range(C*ZC)] for j in range(R*ZR)]
for i in range(R):
    for j in range(C):
        for k in range(ZR*i,ZR*(i+1)):
            for m in range(ZC*j,ZC*(j+1)):
                matrix[k][m] = article[i][j]
print('\n'.join(''.join(row) for row in matrix))

File: skener.java
import java.util.*;

public class skener {
    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int[] intArray = Arrays.stream(s.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        int R = intArray[0], C = intArray[1], ZR = intArray[2], ZC = intArray[3];
        String[][] matrix = new String[R*ZR][C*ZC];
        for (int i = 0; i < R; i++){
            String line = s.nextLine().strip();
            for (int j = 0; j < C; j++){
                String symbol = line.substring(j,j+1);
                for (int k = ZR*i; k < ZR*i + ZR; k++){for (int m = ZC*j; m < ZC*j + ZC; m++){matrix[k][m] = symbol;}}
            }
        }
        for (String[] row: matrix){
            for (String symbol: row){System.out.print(symbol);}
            System.out.println(); //new line
        }
        s.close();
    }
}
